lib/Service/SocialApiService.php

```diff
@@ -124,6 +141,22 @@ protected function getAddressBook(string $addressbookId) : ?IAddressBook {
 	}
 
 
+	/**
+	 * @NoAdminRequired
+	 *
+	 * Retrieves and initiates all addressbooks from a user
+	 *
+	 * @param {string} user the user to query
+	 * @param {IManager} the contact manager to load
+	 */
+	protected function registerAddressbooks($user, IManager $manager) {
+		$cm = new ContactsManager($this->davBackend, $this->l10n);
+		$cm->setupContactsProvider($manager, $user, $this->urlGen);
```

Comment:
Is there a way to limit this to address books owned by a user (no system address book, no shared address books from other users)? Or, alternatively, is there a way to find out about an owner of an IAddressBook?

Reply:
cc @georgehrke what do you think? Does it make sense to add filters to the IAddressBook interface?